Assets/GameAssets/World/WorldScripts/Interactables/TestInteractable.cs

```diff
@@ -0,0 +1,27 @@
+/**********************************************************************************************************************
+// File Name :         TestInteractable.cs
+// Author :            Alex Kalscheur
+// Creation Date :     11/13/24
+// 
+// Brief Description : Simple test script for use in InteractableTestGym
+**********************************************************************************************************************/
+
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+/// <summary>
+/// class for interactable test
+/// </summary>
+public class TestInteractable : MonoBehaviour, IPlayerInteractable
```

Comment:
Since this is a test, don't forget to remove it when you are done with it.

Reply:
Should it be deleted if it can still be used in the test gym for future interactables? /genq

Reply:
Good question, not entirely certain, but I personally would delete it.

Reply:
always delete now then bring back later since it will always be saved in github somewhere